unattended_installer/install_functions/opendistro/checks.sh

```diff
@@ -133,11 +136,19 @@ function checkArguments() {
 
     # -------------- Cluster start ----------------------------------
 
-    if [[ -n "${start_elastic_cluster}" && ( -n "${AIO}" || -n "${elasticsearch}" || -n "${kibana}" || -n "${wazuh}" || -n "${overwrite}" || -n "${configurations}" || -n "${tar_conf}" || -n "${uninstall}") ]]; then
-        logger -e "The argument -s|--start-cluster can't be used with -a, -k, -e or -w arguments."
+    if [[ -n "${start_elastic_cluster}" && ( -z "${elasticsearchinstalled}" || -z "${elastic_remaining_files}") ]]; then
+        logger -e "The argument -s|--start-cluster need elasticsearch installed. Run the script with the parameter first --elasticsearch <elasticsearch-node-name>."
         exit 1
+    else
+        if [[ -n "${start_elastic_cluster}" && ( -n "${AIO}" || -n "${elasticsearch}" || -n "${kibana}" || -n "${wazuh}" || -n "${overwrite}" || -n "${configurations}" || -n "${tar_conf}" || -n "${uninstall}") ]]; then
+            logger -e "The argument -s|--start-cluster can't be used with -a, -k, -e or -w arguments."
+            exit 1
+        fi
```

Comment:
This `if else` must be 2 different ifs. 
In addition to this, it this function is before to `checkIfInstalled`, `elastic_remaining_files` is not used yet, right?

Reply:
Done, and fixed